access/resource_permissions_test.go

```diff
@@ -419,3 +460,86 @@ func TestResourcePermissionsCreate_error(t *testing.T) {
 		}
 	}
 }
+
+func TestAccAddOrModifyDeleteJobPermissions(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	randomName := qa.RandomName()
+
+	if cloudEnv == "" {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+
+	client := common.CommonEnvironmentClient()
+
+	jobSettings := compute.JobSettings{
+		NewCluster: &compute.Cluster{
+			NumWorkers:   2,
+			SparkVersion: "6.4.x-scala2.11",
+			SparkConf:    nil,
+			NodeTypeID:   "Standard_DS3_v2",
+		},
+		Name: "Job " + randomName,
+	}
+	jobCreate, err := compute.NewJobsAPI(client).Create(jobSettings)
+
+	assert.NoError(t, err)
+
+	jobID := fmt.Sprint(jobCreate.ID())
+
+	groupName := "Group" + randomName
+	_, groupsAPIErr := identity.NewGroupsAPI(client).Create(groupName, nil, nil, nil)
+
+	assert.NoError(t, groupsAPIErr)
+
+	userName := randomName + "@" + randomName + ".com"
```

Comment:
security bug: what if this random domain name is actually registered by hackers and they gain access to workspace? replace it with `example.com`, which is guaranteed to be ignoring all mails.